tests/utils/test_databricks_utils.py

```diff
@@ -0,0 +1,12 @@
+from mlflow.utils import databricks_utils
+
+
+def test_no_throw():
+    """
+    Outside of Databricks the databricks_utils methods should never throw and should only return
+    None.
+    """
+    assert not databricks_utils.is_in_databricks_notebook()
+    assert databricks_utils.get_notebook_id() is None
```

Comment:
One question, why don't we want the Databricks methods to throw when running outside of Databricks? It looks like we guard their use by checking `is_in_databricks_notebook` in `start_run`